src/communication/xcSession.ts

```diff
@@ -32,6 +32,8 @@ export class DefaultSession implements Session {
     private publishers: Array<Publisher>;
     private subscribers: Array<Subscriber>;
     private privateTopics: Array<String>;
+    private heartBeatId: NodeJS.Timer;
```

Comment:
To stop sending heartbeat, we must stop the setInterval function by doing         clearInterval(heartbeatId). It is done when the session is closed.